common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4628,6 +4628,12 @@
     ],
     "sqlState" : "42K09"
   },
+  "TUPLE_SIZE_EXCEEDS_LIMIT" : {
+    "message" : [
+      "Due to Scala's limited support of tuple, tuple with more than 22 elements are not supported."
```

Comment:
Users can face to the error not only for wide tuples, see:
```
      if (numElements < 1 || numElements > MAX_TUPLE_ELEMENTS) {
        throw ExecutionErrors.elementsOfTupleExceedLimitError()
      }
```
but for empty tuples too. I think it would be nice to add a separate error condition for the case.

Reply:
Sounds good. Let me address it

Reply:
Updated. Thanks!